tests/cacl/test_cacl_application.py

```diff
@@ -823,12 +821,12 @@ def test_cacl_application_dualtor(duthost_dualtor, tbinfo, localhost, creds, doc
     """
     verify_cacl(duthost_dualtor, tbinfo, localhost, creds, docker_network, expected_dhcp_rules_for_standby)
 
-def test_multiasic_cacl_application(duthosts, tbinfo, rand_one_dut_hostname, localhost, creds,docker_network, enum_frontend_asic_index):
+def test_multiasic_cacl_application(duthosts, tbinfo, rand_one_dut_hostname, localhost, creds, docker_network, expected_dhcp_rules_for_standby, enum_frontend_asic_index):
```

Comment:
@wenyiz2021  Thank you for your hard work to fix this issue. I think your solution is better, should we add `expected_dhcp_rules_for_standby` for `test_cacl_application_nondualtor` as well, even it works fine based on current code.
One wired thing is in the traceback of test_cacl_application_nondualtor, `expected_dhcp_rules_for_standby` is 1, that's the reason why this case failed. I verified `test_cacl_application_nondualtor`, the `expected_dhcp_rules_for_standby` is null. What's the difference for these 2 cases, I didn't figure out.

Reply:
@ZhaohuiS I didn't change for test_cacl_application_nondualtor, because it is calling verify_cacl this way:
verify_cacl(duthost, localhost, creds, docker_network, None)
where def is:
def verify_cacl(duthost, localhost, creds, docker_network, expected_dhcp_rules_for_standby, asic_index = None)

so it is passing None to expected_dhcp_rules_for_standby, and leave asic_index = None, so it should be good.

may I know where you find "traceback of test_cacl_application_nondualtor, expected_dhcp_rules_for_standby is 1"?

Reply:
Alternatively, I gave default None to expected_dhcp_rules_for_standby at verify_cacl() definition.

So that only test_multiasic_cacl_application that called verify_cacl with an asic_index need to specifically pass None to expected_dhcp_rules_for_standby. 
test_cacl_application_dualtor can pass its expected_dhcp_rules_for_standby.

Other callers don't need to specify for the 2 arguments.

Reply:
@wenyiz2021 I got it, this the difference, if I put expected_dhcp_rules_for_standby behind asic_index, it should be fine. As you said, it's correct to set it to None. Thank you for fixing this.

 the log of "traceback of test_cacl_application_nondualtor, expected_dhcp_rules_for_standby is 1" is [here](https://dev.azure.com/mssonic/internal/_build/results?buildId=134104&view=logs&j=bed026c5-b179-58e8-339f-77f4b7309744&t=3d56b420-5b0a-59fb-e15f-64a2477dcafc&l=25633):

Reply:
@ZhaohuiS yeah, that traceback comes from test_multiasic_cacl_application(), where the integer comes from asic_index iso intending for expected_dhcp_rules_for_standby .